command/runner.go

```diff
@@ -203,3 +223,69 @@ func (r *cmdWaitReadCloser) Close() error {
 	r.closed = true
 	return nil
 }
+
+// Close the pipe and wait until the consumer program exits
+func (p pagerPipe) Close() error {
+	err := p.writer.Close()
+	<-*p.cc
+	return err
+}
+
+func (p pagerPipe) Write(b []byte) (int, error) {
+	return p.writer.Write(b)
+}
+
+func runPager(pagerCmd string) (*pagerPipe, error) {
+	pager := ""
+	if pagerCmd != "" {
+		pager = pagerCmd
+	} else if p := os.Getenv("KUBECOLOR_PAGER"); p != "" {
+		pager = p
+	} else if p := os.Getenv("PAGER"); p != "" {
```

Comment:
I think you can remove these `Getenv` as our config also loads the value from env vars.

In viper, you can use `BindEnv` to also make `pager` config field listen to `PAGER` env var. I think we're using that already on some fields

Reply:
Nice catch! Yep fully agree.

Could we have a test on that, to make sure we don't cause a regression on that?

Or at least a comment that explains why it isn't using `viper.BindEnv`

Reply:
I added a test and a comment